riscv/triggers.h

```diff
@@ -228,12 +235,22 @@ class mcontrol_t : public mcontrol_common_t {
 public:
   virtual reg_t tdata1_read(const processor_t * const proc) const noexcept override;
   virtual void tdata1_write(processor_t * const proc, const reg_t val, const bool allow_chain) noexcept override;
+
+  virtual void set_hit(hit_t UNUSED val) override { hit = true; }
```

Comment:
Seems safer to do: `hit = val != HIT_FALSE;`

Reply:
The hit variable has the type hit_t, which cannot be assigned with a bool value.